src/server/ua_services_attribute.c

```diff
@@ -411,9 +411,7 @@ void Service_Read(UA_Server *server, UA_Session *session, const UA_ReadRequest *
 		size_t offset = 0;
         UA_Connection *c = UA_NULL;
         UA_SecureChannel *sc = session->channel;
-        if(sc)
```

Comment:
No, I was a little unsure about this one. I couldn't find any reference to `sc` off session in the history so it looked a bit out of place. I probably should have left it as is but was trying to get it all compiling. Happy to revert, unless you can offer some guidance?